qubes-dist-upgrade.sh

```diff
@@ -142,6 +139,14 @@ restore_rpmsave_policy() {
     done
 }
 
+has_updated_qubes_version() {
+    if grep -q 'VERSION_ID=4.2' /etc/os-release; then
+        true
+    else
+        false
+    fi
```

Comment:
just the `grep -q` line? the if+true+false part is redundant

Reply:
yes, it was some pending code to know if more checks were needed or not like checking also xen check maybe?

Reply:
In all the cases, it can be a one line indeed.